packages/api-client/src/__tests__/mockConnections/MockAuthApi.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Had to add this to the top of all the mock apis since they don't implement many of the methods